react/features/filmstrip/components/web/BaseIndicator.js

```diff
@@ -27,7 +27,7 @@ class BaseIndicator extends Component {
         iconClassName: React.PropTypes.string,
 
         /**
-         * The front size for the icon.
+         * The font size for the icon.
```

Comment:
typo I found while fixing other stuff. Unrelated but trivial so I'm trying to "sneak it in."